documentation/modules/auxiliary/gather/roundcube_auth_file_read.md

```diff
@@ -0,0 +1,165 @@
+## Vulnerable Application
+
+Roundcube Webmail allows unauthorized access to arbitrary files on the host's filesystem, including configuration files.
+This affects all versions from 1.1.0 through version 1.3.2. The attacker must be able to authenticate at the target system
+with a valid username/password as the attack requires an active session.
+
+Tested against version 1.3.2
+
+### Install Roundcube 1.3.2 on Ubuntu 22.04
+
+Instructions are loosely based on https://www.digitalocean.com/community/tutorials/how-to-install-your-own-webmail-client-with-roundcube-on-ubuntu-16-04
+
+The main point of pain is installing PHP 7.0 on Ubuntu 22.04
+
+#### Install LAMP
+
+```
+sudo apt-get install -y tasksel
+sudo tasksel install lamp-server
+```
+
+#### Install PHP 7.0
+
+```
+sudo apt install software-properties-common ca-certificates lsb-release apt-transport-https dbconfig-sqlite3 php7.0-sqlite3
+LC_ALL=C.UTF-8 sudo add-apt-repository ppa:ondrej/php
+sudo apt update
+sudo apt-get install php7.0 php7.0-xml php7.0-mbstring php7.0-intl php7.0-zip
+sudo a2dismod php8.1
+sudo a2enmod php7.0
+```
+
+#### Configure PHP
+
+```
+sudo nano /etc/php/7.0/apache2/php.ini
+```
+
+Uncomment the following lines:
+
+```
+extension=php_mbstring.dll
+extension=php_xmlrpc.dll
+extension=php_pdo_sqlite.dll
+```
+
+Add the following line to the end of the extension list:
+
+```
+extension=dom.so
+```
+
+Uncomment and change the following values:
+
+```
+date.timezone = "America/New_York"
+upload_max_filesize = 12M
+post_max_size = 18M
+mbstring.func_overload = 0
+```
+
+Save and close the file, now reload Apache
+
+```
+sudo systemctl restart apache2
+```
+
+#### Install dovecot
+
+```
+sudo apt install dovecot-imapd
+```
+
+#### Install Roundcube
+
+```
+wget https://github.com/roundcube/roundcubemail/releases/download/1.3.2/roundcubemail-1.3.2-complete.tar.gz -O /tmp/roundcubemail-1.3.2-complete.tar.gz
+sudo tar -zxf /tmp/roundcubemail-1.3.2-complete.tar.gz -C /var/www/html/
+sudo chown -R root:root /var/www/html/roundcubemail-1.3.2/
+```
+
```

Comment:
#### Configure Apache
```
sudo cp /etc/apache2/sites-available/000-default.conf /etc/apache2/sites-available/roundcubemail-1.3.2-complete.config
sudo vi /etc/apache2/sites-available/roundcubemail-1.3.2-complete.config
```

Update
`ServerName <rhost IP>`
`DocumentRoot /var/www/html/roundcubemail-1.3.2/`

Save and close the file, now reload Apache

```
sudo systemctl restart apache2
```